config/default.yml

```diff
@@ -614,6 +614,15 @@ RSpec/NestedGroups:
   VersionChanged: '1.10'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NestedGroups
 
+RSpec/NoExpectationExample:
+  Description: Checks if an example includes any expectation.
+  Enabled: pending
+  ExampleMethodNames: []
```

Comment:
We have a "global" setting for this https://github.com/rubocop/rubocop-rspec/blame/master/config/default.yml#L36